src/test/resources/pages/instructorFeedbackAddSuccess.html

```diff
@@ -1645,37 +1645,53 @@ <h5 class="col-sm-2">
           </div>
         </div>
         <br>
-        <div class="col-sm-12 padding-15px background-color-light-green">
+        <div class="col-sm-12 margin-bottom-15px padding-15px background-color-light-green">
           <div class="col-sm-12 padding-0 margin-bottom-7px">
             <b class="visibility-title">
               Visibility
             </b>
             (Who can see the responses?)
           </div>
-          <div class="col-sm-6 btn-group" data-toggle="buttons">
-            <label class="btn btn-xs btn-info visibilityOptionsLabel" id="visibilityOptionsLabel--1" onchange="toggleVisibilityEditTab(this)">
-              <input type="radio">
-              <span class="glyphicon glyphicon-pencil">
-              </span>
-              Edit Visibility
-            </label>
-            <label class="btn btn-xs btn-info active visibilityMessageButton" id="visibilityMessageButton--1" onchange="toggleVisibilityPreviewTab(this)">
-              <input type="radio">
-              <span class="glyphicon glyphicon-eye-open">
+          <div class="visibility-options-dropdown btn-group col-sm-12 margin-bottom-15px background-color-light-green">
```

Comment:
`background-color-light-green' is likely no longer needed

also check if other classes are still needed